src/BooleanSlider/widget/BooleanSlider.js

```diff
@@ -117,8 +117,8 @@ define([
             if (this._contextObj !== null) {
                 dojoStyle.set(this.domNode, "display", "block");
 
-                dojoProp.set(this.trueNode, "textContent", this.trueValue);
-                dojoProp.set(this.falseNode, "textContent", this.falseValue);
```

Comment:
Ah. That explains a lot. This is a false-positive unfortunately.

In the past some used ``mxui.dom.textContent``, which is now replaced by ``mxui.dom.text``. The deprecation-checker falsely said this is a deprecation, which is not :-)